### ISSUE TITLE
strapi-helper-plugin style guide

### ISSUE THREAD

**Issue by moschan** [strapi/strapi#6876]
- [x] **I have created my request on the Product Board before I submitted this issue**
- [x] **I have looked at all the other requests on the Product Board before I submitted this issue**

**Please describe your feature request:**

If you want to extend the function, you will create own plugin. strapi provides some shared helper component which you can use for building UI of plugin. I think it's better to use those components as much as possible to make components can provide same experience.
Currently we don't have style guide of strapi-helper-plugin, so it's hard to use those components. If there is style guide, developers can easily find component which developer wants.

**Comment by soupette** [comment]
Hello @moschan the `strapi-helper-plugin` is a package where components will be deprecated (the migration takes time so we continue to use it a bit) but the idea is to use [Buffet.js](https://buffetjs.io/) here's the associated [documentation](https://buffetjs.io/storybook/?path=/story/get-started--welcome). At some point the administration will only use very specific components that are only made for strapi and all the files that are located in both the [utils folder](https://github.com/strapi/strapi/tree/master/packages/strapi-helper-plugin/lib/src/utils) and the [hooks one](https://github.com/strapi/strapi/tree/master/packages/strapi-helper-plugin/lib/src/hooks). Usually when you create a plugin the main utility you want to use is the [request helper](https://github.com/strapi/strapi/blob/master/packages/strapi-helper-plugin/lib/src/utils/request.js). We will provide some documentation soon!

**Comment by moschan** [comment]
Thanks @soupette I understand current situation.

Then I think it's better that:

- Adding notice here (https://github.com/strapi/strapi/tree/master/packages/strapi-helper-plugin) to prevent developer use those components to create new plugin.
- Adding link and guide here (https://strapi.io/documentation/v3.x/plugin-development/frontend-development.html) for making developer use Buffet.js as much as possible.

Am I correct? If so I'm very happy to make PR for it.

**Comment by soupette** [comment]
Yes exactly they should only use the utils from this package and the `PopUpWarning` component.

**Comment by moschan** [comment]
Thanks, I gonna work on it.

**Comment by lain1936** [comment]
![image](https://user-images.githubusercontent.com/16677117/88259450-e9833c80-ccf4-11ea-8983-ecf5242b2bc6.png)
when I change the  height of  the header bar,  It seems  that  the HeaderSearch/Wrapper in  strapi-helper-plugin used by content-manager listview  cannot be directly overwrite
